upgrade/upgrade-scripts/upgrade-container

```diff
@@ -815,18 +815,8 @@ function migrate_configuration() {
 }
 
 function do_upgrade_container_in_place() {
-	[[ -f "/var/lib/delphix-appliance/platform" ]] ||
-		die "could not determine platform; file does not exist"
-
-	run "$IMAGE_PATH/execute" \
-		-p "$(cat /var/lib/delphix-appliance/platform)" ||
+	run "$IMAGE_PATH/execute" ||
 		die "'$IMAGE_PATH/execute' failed in '$CONTAINER'"
-
-	run /bin/systemctl reload delphix-platform ||
-		die "'systemctl reload delphix-platform' failed in '$CONTAINER'"
-
-	run /bin/systemctl restart delphix-platform ||
-		die "'systemctl restart delphix-platform' failed in '$CONTAINER'"
```

Comment:
Do we not need these because `execute` will restart `delphix.target` one way or the other?

Reply:
yea, that's the idea.. we might want/need to reload delphix-platform though.. that's a bit awkward, but I don't think restarting delphix.target will do that for us.. might need to do something like that via the app-gate, too..

Reply:
been awhile since I wrote this code.. just wanted to follow up.. we (now) reload delphix-platform via the `execute` script via these changes.. so I think we're good.